index.html

```diff
@@ -0,0 +1,161 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Lesson 1</title>
+</head>
+<body>
+<script type="text/javascript">
+    'use strict'
+    // 1.
+    console.log('Задание 1');
+    const loop = function (times = 0, callback = null) {
+        if (typeof (callback) == 'function') {
+            for (let i = 0; i < times; i++) {
+                callback();
+            }
+        } else {
+            console.log('Callback не функция')
+        }
+    };
+    let a = 2;
+    console.log(loop(a, 10));
+    console.log(loop(5, function sum() {
+        console.log(a++)
+    }));
+    console.log(loop(10, function hello() {
+        console.log('Hello, world')
+    }));
+
+    // 2.
+    console.log('Задание 2');
+
+    function calculateArea(figure, b, c) {
+        let [, ...input] = arguments;
+        let area;
+        if (b && c !== 0) {
```

Comment:
```suggestion
        if (b !== 0 && c !== 0) {
```
`или if(b && c)`
:)